res/controllers/engine-api.d.ts

```diff
@@ -301,4 +301,57 @@ declare namespace engine {
      *               SoftStart with low factors would take a while until sound is audible. [default = 1.0]
      */
     function softStart(deck: number, activate: boolean, factor?: number): void;
+
+    enum WellKnownCharsets {
+        Latin1,
+        ISO_8859_1,
+        Latin9,
+        ISO_8859_15,
+        UCS2,
+        ISO_10646_UCS_2
+    }
+
+    /**
+     * Converts a string into another charset.
+     *
+     * This function is useful to display text on a device that does not make use of UTF-8.
+     * Available charset names are listed here: http://www.iana.org/assignments/character-sets/character-sets.xhtml
+     *
+     * Depending on the system you run on, this function may convert characters that are unsupported by the target
+     * charset to either the codepoint for '?' or the substitute character.
+     * @see https://en.wikipedia.org/wiki/Substitute_character
+     *
+     * Example usage:
+     *
+     *         const group = "…"  // device midi group
+     *         const titleLength = engine.getValue(group, "track_title_length")
+     *         let title = ""
+     *         for (let i = 0; i < titleLength; i++) {
+     *             title += String.fromCodePoint(engine.getValue(group,`track_title_${i}`))
```

Comment:
I'm not sure `String.fromCodePoint` exists in `QJSEngine`. [MDN states that it is widely available](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint) my IDE, however, didn't know of it.